Resources/views/change_password/form.html.twig

```diff
@@ -0,0 +1,4 @@
+{{ form_start(form, {'method': 'post', 'action': path('webstack_user_change_password_index'), 'attr': {'class': 'webstack_user_change_password'}}) }}
```

Comment:
Ik denk dat je de action het beste leeg kan laten, dan pakt hij namelijk altijd de huidige pagina waarop het formulier wordt ingeladen. Als er dan vanuit een project een custom controller wordt gemaakt dan wordt die ook aangeroepen en niet forced degene in de bundle.

Reply:
En waarom een css class?

Reply:
en als al die dingen niet nodig zijn dan is zelfs een `{{ form(form)}}` voldoende